### DISCUSSION TITLE
Is there a pre-trained model for text recognition based on mmocr on a large Chinese dataset?

### DISCUSSION THREAD

**Discussion by vansin** [open-mmlab/mmocr#1464] [Q&A]
Is there a pre-trained model for text recognition based on mmocr on a large Chinese dataset?

有基于mmocr 在大规模中文数据集上的  文本识别预训练模型吗？

**Comment by htaoruan** [comment]
i need it  thanks

**Comment by gaotongxiao** [comment]
There's one in MMOCR 0.6: https://mmocr.readthedocs.io/en/latest/textrecog_models.html#chinese-dataset
